packages/cashier/src/Sass/account-transfer-form.scss

```diff
@@ -0,0 +1,175 @@
+.account-transfer-form {
+    &__hint {
+        color: var(--text-general);
+    }
+    &__submit-button {
+        @include desktop {
+            margin: 3.4rem 0 2.4rem;
+        }
+        @include mobile {
+            margin: 2.4rem 0;
+        }
+    }
+    &__crypto {
+        &--percentage-selector {
+            display: flex;
+            flex-direction: column;
+            align-items: center;
+            margin-bottom: 2.4rem;
+
+            & .dc-text {
+                margin-top: 0.4rem;
+            }
+        }
+    }
+    &__drop-down-wrapper {
+        @include mobile {
+            margin-top: 1.2rem;
+
+            .dc-dropdown__display {
+                min-height: 4rem !important;
+            }
+        }
+    }
+    &__drop-down {
+        display: inline-block;
+        min-width: 20.3rem;
+        min-height: 4rem;
+        line-height: 1.43;
+        text-align: left;
```

Comment:
You can add these props with the help of the `Text` component.

Reply:
These are used by `Dropdown` component.